proposals/extensions/serialization.md

```diff
@@ -67,38 +68,38 @@ The following interfaces are provided in the library to be implemented by the se
 
 ```kotlin
 interface SerializationStrategy<in T> {
-    fun serialize(output: Encoder, obj : T)
+    fun serialize(encoder: Encoder, value: T)
     val descriptor: SerialDescriptor
```

Comment:
Here it also would be nice to comply our own code style and have vals before functions